### PR TITLE
Adding apps3

### PR DESCRIPTION
**Pull request by RodrigoVanzelotti**
Just did my last modifications to the app, until creating-and-updating-figures.
I dont have any exp with the bad apple example, so I will leave it as its, please someone fix it.

### CODE REVIEW

File: pages/3d-scatter-plots.py [hunk 1]
```diff
@@ -4,17 +4,19 @@
...
 
 
 notes = """
+#### Dash Components in the App:
+- [RangeSlider](https://plotly.com/python/range-slider/)
 
 #### Plotly documentation:  
-
 - [3d Scatter Plots](https://plotly.com/python/3d-scatter-plots/)
+- [Regular Callbacks](https://dash.plotly.com/basic-callbacks)
```
Comment (Coding-with-Adam) [inline pages/3d-scatter-plots.py:19]:
```suggestion
```

File: pages/3d-scatter-plots.py [hunk 2]
```diff
@@ -4,16 +4,17 @@
...
-    __name__, description="Iris samples filtered by petal width with a slider"
+    __name__, description="Iris samples in a scatter_3d plot filtered by petal width with a RangeSlider. This app has a top-bottom layout and a regular-callback."
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in the App:
+- [RangeSlider](https://plotly.com/python/range-slider/)
```
Comment (Coding-with-Adam) [inline pages/3d-scatter-plots.py:15]: This rangeslider is a dash core component. Can you please update the link to the rangeslider component in the dash docs. https://dash.plotly.com/dash-core-components/rangeslider

File: pages/animations.py [hunk 3]
```diff
@@ -20,6 +20,7 @@
 - [Intro to animations](https://plotly.com/python/animations/)
 - [Bar Plot](https://plotly.com/python/bar-charts/)
 - [Scatter Plot](https://plotly.com/python/line-and-scatter/)
+- [Regular Callbacks](https://dash.plotly.com/basic-callbacks)
```
Comment (Coding-with-Adam) [inline pages/animations.py:23]: To be removed.

File: pages/axes.py [hunk 4]
```diff
@@ -5,21 +5,23 @@
...
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```
Comment (Coding-with-Adam) [inline pages/axes.py:17]: I can't see a dropdown in the code. But I saw a button. Can you please replace this link with the link to the html.button. https://dash.plotly.com/dash-html-components/button

File: pages/axes.py [hunk 5]
```diff
@@ -5,21 +5,23 @@
...
 
+#### Plotly Documentation:  
 - [Adjusting the axes properties](https://plotly.com/python/axes/)
+- [Bar Plot](https://plotly.com/python/bar-charts/)
```
Comment (Coding-with-Adam) [inline pages/axes.py:21]: although it looks like a bar plot, the code is actually using a px.histogram. Can you please replace this.

File: pages/axes.py [hunk 6]
```diff
@@ -5,21 +5,23 @@
...
 
 Learn how to adjust axes properties in Python - axes titles, styling and coloring axes and grid lines, ticks, tick labels and more.
```
Comment (Coding-with-Adam) [inline pages/axes.py:23]: I think this sentence needs to be removed because it doesn't belong here.

File: pages/bar-charts.py [hunk 7]
```diff
@@ -5,18 +5,19 @@
...
 notes = """
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
 #### Plotly Documentation:  
-
-- [How to make Bar Charts](https://plotly.com/python/bar-charts/)
-
+- [Bar Plots](https://plotly.com/python/bar-charts/)
+- [Regular Callbacks](https://dash.plotly.com/basic-callbacks)
```
Comment (Coding-with-Adam) [inline pages/bar-charts.py:20]: To be removed.

File: pages/bar-charts.py [hunk 8]
```diff
@@ -5,18 +5,19 @@
...
-    description="Bar Chart with a Dropdown",
+    description="Bar Chart with a Dropdown. This app has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```
Comment (Coding-with-Adam) [inline pages/bar-charts.py:16]: Let's call it just Dropdown. Please remove the dcc part.

File: pages/box-plots.py [hunk 9]
```diff
@@ -5,17 +5,20 @@
...
+    description="Interactive box plot with Checklist and RadioItems. It has a top-bottom layout and a regular-callback",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
+- [Dcc.Checklist](https://dash.plotly.com/dash-core-components/checklist)
```
Comment (Coding-with-Adam) [inline pages/box-plots.py:17]: Let's call it just RadioItems and Checklist. Please remove the dcc part.

File: pages/box-plots.py [hunk 10]
```diff
@@ -5,17 +5,20 @@
...
 
 #### Plotly Documentation:  
+- [Box Plots](https://plotly.com/python/box-plots/)
 
-- [How to make box plots](https://plotly.com/python/box-plots/)
 A box plot is a statistical representation of the distribution of a variable through its quartiles. 
 The ends of the box represent the lower and upper quartiles, while the median (second quartile) is marked by a line inside the box.
```
Comment (Coding-with-Adam) [inline pages/box-plots.py:23]: @RodrigoVanzelotti let's remove this explanation. I want to make sure it's consistent with other files in the Pages folder. None of the other files have an explanation inside the notes sections.

File: pages/builtin-colorscales.py [hunk 11]
```diff
@@ -3,16 +3,18 @@
...
 
-dash.register_page(__name__, description="Set the color scale with a Dropdown")
+dash.register_page(__name__, description="Set the color scale with a Dropdown. It has a top-bottom layout and a regular-callback")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```
Comment (Coding-with-Adam) [inline pages/builtin-colorscales.py:13]: remove dcc. Just keep Dropdown

File: pages/builtin-colorscales.py [hunk 12]
```diff
@@ -3,16 +3,18 @@
...
 
 #### Plotly Documentation:  
-
 - [Continuous Color Scales and Color Bars](https://plotly.com/python/colorscales/)
+
 How to set, create and control continuous color scales and color bars in scatter, bar, map and heatmap figures.
```
Comment (Coding-with-Adam) [inline pages/builtin-colorscales.py:18]:
```suggestion
```

File: pages/builtin-colorscales.py [hunk 13]
```diff
@@ -3,17 +3,18 @@
...
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
 #### Plotly Documentation:  
-
 - [Continuous Color Scales and Color Bars](https://plotly.com/python/colorscales/)
-How to set, create and control continuous color scales and color bars in scatter, bar, map and heatmap figures.
+
```
Comment (Coding-with-Adam) [inline pages/builtin-colorscales.py:17]: This app uses the scatter plot also.
@RodrigoVanzelotti can you please add a link to the plotly docs scatter Plot, under the Continuous Color Scales link.

**Review by Coding-with-Adam — CHANGES_REQUESTED** [review]
Nice PR. Thanks @RodrigoVanzelotti . I added a few changes that need to be made. 
I've reviewed up to the builtin colorscales file. Tomorrow, I'll continue the review from the candlestick-charts to the end. 
But much of what I requested in this first review can be implemented in the remaining files I haven't looked at yet. For examples, remove regular-callback links if you added them, and call the components by their name instead of dcc.Dropdown or dcc.RadioItems, etc.

File: pages/candlestick-charts.py [hunk 14]
```diff
@@ -3,16 +3,19 @@
...
+dash.register_page(__name__, description="Candlestick chart with a range slider. This app has a top-bottom layout and a regular-callback.")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Checklist](https://dash.plotly.com/dash-core-components/checklist)
 
 #### Plotly Documentation:
```
Comment (Coding-with-Adam) [inline pages/candlestick-charts.py:16]:
```suggestion
```

File: pages/candlestick-charts.py [hunk 15]
```diff
@@ -3,16 +3,18 @@
...
 
 notes = """
+#### Dash Components in App:  
+- [Checklist](https://dash.plotly.com/dash-core-components/checklist)
 
 #### Plotly Documentation:  
-
 - [Candlestick charts](https://plotly.com/python/candlestick-charts/)
+- [RangeSlider](https://plotly.com/python/range-slider/)
 How to make interactive candlestick charts in Python with Plotly. Six examples of candlestick charts with Pandas, time series, and yahoo finance data.
```
Comment (Coding-with-Adam) [inline pages/candlestick-charts.py:18]:
```suggestion
```

File: pages/choropleth-maps.py [hunk 16]
```diff
@@ -4,17 +4,19 @@
 
 
 dash.register_page(
-    __name__, description="Interactive Mapbox Choropleth Map of US Counties"
+    __name__, description="Interactive Mapbox Choropleth Map of US Counties. This app has a top-bottom layout and a regular-callback."
```
Comment (Coding-with-Adam) [inline pages/choropleth-maps.py:7]:
```suggestion
    __name__, description="Interactive Mapbox Choropleth Map of Montreal. This app has a top-bottom layout and a regular-callback."
```

File: pages/choropleth-maps.py [hunk 17]
```diff
@@ -4,17 +4,19 @@
...
-    __name__, description="Interactive Mapbox Choropleth Map of US Counties"
+    __name__, description="Interactive Mapbox Choropleth Map of Montreal. This app has a top-bottom layout and a regular-callback."
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
```
Comment (Coding-with-Adam) [inline pages/choropleth-maps.py:15]:
```suggestion
- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
```

File: pages/choropleth-maps.py [hunk 18]
```diff
@@ -4,17 +4,19 @@
...
 
 notes = """
+#### Dash Components in App:  
+- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
 
 #### Plotly Documentation:  
-
 - [Choropleth maps](https://plotly.com/python/mapbox-county-choropleth/)
+
 How to make an interactive Mapbox Choropleth Map of US Counties.
```
Comment (Coding-with-Adam) [inline pages/choropleth-maps.py:20]:
```suggestion
- [Choropleth maps](https://plotly.com/python/choropleth-maps/)

```

File: pages/creating-and-updating-figures.py [hunk 19]
```diff
@@ -3,15 +3,18 @@
...
-dash.register_page(__name__, description="Updating the figure layout")
+dash.register_page(__name__, description="Updating the figure layout using a Slider, and an Input. This app has a top-bottom layout and a regular-callback.")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.Input](https://dash.plotly.com/dash-core-components/input)
+- [Dcc.Slider](https://dash.plotly.com/dash-core-components/slider)
```
Comment (Coding-with-Adam) [inline pages/creating-and-updating-figures.py:14]:
```suggestion
dash.register_page(__name__, description="Updating the figure's title's font size and text, using a Slider and an Input. This app has a top-bottom layout and a regular-callback.")

filename = __name__.split("pages.")[1]


notes = """
#### Dash Components in App:  
- [Input](https://dash.plotly.com/dash-core-components/input)
- [Slider](https://dash.plotly.com/dash-core-components/slider)
```

File: pages/creating-and-updating-figures.py [hunk 20]
```diff
@@ -3,15 +3,18 @@
...
 
 notes = """
+#### Dash Components in App:  
+- [Input](https://dash.plotly.com/dash-core-components/input)
+- [Slider](https://dash.plotly.com/dash-core-components/slider)
 
 #### Plotly Documentation:  
-
+- [Sliders](https://plotly.com/python/sliders/)
 - [Creating and updating figures](https://plotly.com/python/creating-and-updating-figures)
```
Comment (Coding-with-Adam) [inline pages/creating-and-updating-figures.py:19]:
```suggestion
- [Creating and updating figures](https://plotly.com/python/creating-and-updating-figures)
```

File: pages/getting-started.py [hunk 21]
```diff
@@ -4,20 +4,19 @@
...
 
 filename = __name__.split("pages.")[1]
 
 notes = """
 
-#### Plotly Documentation:  
-
-- [Getting Started with Dash and Plotly](https://plotly.com/python/getting-started/)
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```
Comment (Coding-with-Adam) [inline pages/getting-started.py:15]:
```suggestion
- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```

File: pages/getting-started.py [hunk 22]
```diff
@@ -4,20 +4,19 @@
...
-
-- [Getting Started with Dash and Plotly](https://plotly.com/python/getting-started/)
+#### Dash Components in App:  
+- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
 #### Plotly Documentation:  
-
-- [Dash Tutorial](https://dash.plotly.com/installation)
+- [Bar Plots](https://plotly.com/python/bar-charts/)
+- [Regular Callbacks](https://dash.plotly.com/basic-callbacks)
```
Comment (Coding-with-Adam) [inline pages/getting-started.py:21]:
```suggestion
- [Getting started with Plotly](https://plotly.com/python/getting-started/)

```

**Review by Coding-with-Adam — CHANGES_REQUESTED** [review]
Hi @RodrigoVanzelotti 
I just finished updating all the files from candlestick-charts to the end. So, all that is left is to implement the updates from 3d-scatter-plots to builtin-colorscale.
Thank you

**Comment by IcToxi** [comment]
@Coding-with-Adam 
Hi Adam, I noticed that you are revising the description and contributors' info. And I have a suggestion. Would you consider adding the last modified guy's information? I'd love to see the code I submit with some necessary changes to make it more readable and stable, and I've checked Allow Changes when submitting the PR. But I don't think all the changes are necessary, some of the code has changed beyond recognition by this point. Embarrassingly, contributors need to be the title for those stupid changes made before the intent of the original code was fully understood.

**Comment by IcToxi** [comment]
@Coding-with-Adam  Sorry, I meant modifiers. Because some newly added statements are confusing,  combined with the original code, and that's not what the contributors did.

File: pages/animations.py (hunk 3 above)
Comment (Coding-with-Adam) [inline pages/animations.py:23]:
```suggestion
```

File: pages/axes.py (hunk 4 above)
Comment (Coding-with-Adam) [inline pages/axes.py:17]:
```suggestion
- [Button](https://dash.plotly.com/dash-html-components/button)
```

File: pages/axes.py [hunk 23]
```diff
@@ -5,21 +5,23 @@
...
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:  
+- [Button](https://dash.plotly.com/dash-html-components/button)
 
+#### Plotly Documentation:  
 - [Adjusting the axes properties](https://plotly.com/python/axes/)
+- [Bar Plot](https://plotly.com/python/bar-charts/)
```
Comment (Coding-with-Adam) [inline pages/axes.py:21]:
```suggestion
- [Histogram](https://plotly.com/python/histograms)
```

File: pages/axes.py [hunk 24]
```diff
@@ -5,19 +5,20 @@
...
 
-#### Plotly Documentation:  
+#### Dash Components in App:  
+- [Button](https://dash.plotly.com/dash-html-components/button)
 
+#### Plotly Documentation:  
 - [Adjusting the axes properties](https://plotly.com/python/axes/)
-
-Learn how to adjust axes properties in Python - axes titles, styling and coloring axes and grid lines, ticks, tick labels and more.
+- [Histogram](https://plotly.com/python/histograms)
```
Comment (Coding-with-Adam) [inline pages/axes.py:23]:
```suggestion
```

File: pages/bar-charts.py (hunk 8 above)
Comment (Coding-with-Adam) [inline pages/bar-charts.py:16]:
```suggestion
- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```

File: pages/bar-charts.py [hunk 25]
```diff
@@ -5,18 +5,19 @@
...
 notes = """
+#### Dash Components in App:  
+- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
 #### Plotly Documentation:  
-
-- [How to make Bar Charts](https://plotly.com/python/bar-charts/)
-
+- [Bar Plots](https://plotly.com/python/bar-charts/)
+- [Regular Callbacks](https://dash.plotly.com/basic-callbacks)
```
Comment (Coding-with-Adam) [inline pages/bar-charts.py:20]:
```suggestion
```

File: pages/builtin-colorscales.py [hunk 26]
```diff
@@ -3,17 +3,18 @@
...
 
-dash.register_page(__name__, description="Set the color scale with a Dropdown")
+dash.register_page(__name__, description="Set the color scale with a Dropdown. It has a top-bottom layout and a regular-callback")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```
Comment (Coding-with-Adam) [inline pages/builtin-colorscales.py:13]:
```suggestion
- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```

File: pages/box-plots.py (hunk 9 above)
Comment (Coding-with-Adam) [inline pages/box-plots.py:17]:
```suggestion
- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
- [Checklist](https://dash.plotly.com/dash-core-components/checklist)
```

File: pages/box-plots.py [hunk 27]
```diff
@@ -5,17 +5,20 @@
...
+#### Dash Components in App:  
+- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
+- [Checklist](https://dash.plotly.com/dash-core-components/checklist)
 
 #### Plotly Documentation:  
+- [Box Plots](https://plotly.com/python/box-plots/)
 
-- [How to make box plots](https://plotly.com/python/box-plots/)
 A box plot is a statistical representation of the distribution of a variable through its quartiles. 
 The ends of the box represent the lower and upper quartiles, while the median (second quartile) is marked by a line inside the box.
```
Comment (Coding-with-Adam) [inline pages/box-plots.py:23]:
```suggestion
```

File: pages/box-plots.py [hunk 28]
```diff
@@ -5,19 +5,20 @@
...
+#### Dash Components in App:  
+- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
+- [Checklist](https://dash.plotly.com/dash-core-components/checklist)
 
 #### Plotly Documentation:  
+- [Box Plots](https://plotly.com/python/box-plots/)
 
-- [How to make box plots](https://plotly.com/python/box-plots/)
-A box plot is a statistical representation of the distribution of a variable through its quartiles. 
-The ends of the box represent the lower and upper quartiles, while the median (second quartile) is marked by a line inside the box.
```
Comment (Coding-with-Adam) [inline pages/box-plots.py:23]:
```suggestion
```

File: pages/builtin-colorscales.py [hunk 29]
```diff
@@ -3,17 +3,18 @@
...
 
 notes = """
+#### Dash Components in App:  
+- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
 #### Plotly Documentation:  
-
 - [Continuous Color Scales and Color Bars](https://plotly.com/python/colorscales/)
-How to set, create and control continuous color scales and color bars in scatter, bar, map and heatmap figures.
+
```
Comment (Coding-with-Adam) [inline pages/builtin-colorscales.py:17]:
```suggestion
- [Scatter Plot](https://plotly.com/python/line-and-scatter/)
```

**Review by Coding-with-Adam — APPROVED** [review]
All good.

**Pull request merged**